.github/workflows/push-master.yml

```diff
@@ -49,6 +49,7 @@ jobs:
           # (No need to define this secret in the repo settings)
           github_token: ${{ secrets.github_token }}
           build_script_name: build:release:electron
+          release: true
```

Comment:
Not 100% sure I want to switch this on - it means we made one of each merge, and there are a lot.